libraries/consul_service.rb

```diff
@@ -55,6 +55,7 @@ def default_environment
         {
           'GOMAXPROCS' => [node['cpu']['total'], 2].max.to_s,
           'PATH' => '/usr/local/bin:/usr/bin:/bin',
+          'SHELL' => node['consul']['service_shell'],
```

Comment:
Yup! It has a default value of /bin/nologin if present, or /bin/false

https://github.com/poise/poise-service/blob/master/README.md#attributes-2